### ISSUE TITLE
Guideline support on node movement with arrow keys

### ISSUE THREAD

**Issue by hasanbalci** [iVis-at-Bilkent/chise.js#100]
I think, it may be a good idea to add guideline support when nodes are moved with arrow keys. Because, alignment with arrow keys may be easier in some cases.

**Comment by ugurdogrusoz** [comment]
Good idea. @kinimesi can we do this? If so, would it go to the extension or to ChiSE sample app?

**Comment by ugurdogrusoz** [comment]
@kinimesi how much effort is this? straightforward?

**Comment by kinimesi** [comment]
@ugurdogrusoz The main problem is when to clear the guidelines: too quick - user can't see them clearly, little slower - they become annoying (like tooltips).

We will need to listen `position` event (right now, we listen for `drag` event). Since position event is triggered in many cases, without a proper check (which will probably include many corner cases) performance will be effected too.

Do you know any graph editor (or another similar app) which shows guidelines while moving with arrow keys?

**Comment by ugurdogrusoz** [comment]
Let's leave this for future
